1. Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. Найти доверительный интервал для оценки математического ожидания с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256.

Так как известно, что генеральная совокупность распределена нормально с $\sigma=16$, то для расчета доверительного интервала будем использовать $z$-критерий.

$\overline{X} \pm z_{\alpha/2} \cdot \frac{\sigma}{\sqrt{n}}$

$\overline{X} = 80$

$\sigma = 16$, $n = 256$

$\alpha = 0.05$

Из $z$-таблицы $z_{0.025} = 1.96$

Таким образом, искомый доверительный интервал:

$\bigl[80-1.96\cdot \frac{16}{\sqrt{256}} ; 80+1.96\cdot \frac{16}{\sqrt{256}}\bigr]$

$\bigl[80-1.96 ; 80+1.96\bigr]$

$\bigl[78.04 ; 81.96\bigr]$

2. В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные: 6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1. Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95.

Предполагаем, что результаты измерений подчинены нормальному закону распределения, однако среднее квадратичное отклонение нам неизвестно, поэтому для расчета доверительного интервала будем использовать критерий Стьюдента и несмещенную оценку дисперсии по имеющейся выборке.

$\overline{X} \pm t_{\alpha/2} \cdot \frac{\sigma_{н}}{\sqrt{n}}$

In [1]:
import numpy as np

In [4]:
x = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])
x

array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])

In [5]:
x_ = x.mean()
x_

6.590000000000001

In [6]:
sigma = x.std(ddof=1)
sigma

0.4508017549014448

$\alpha=0.05$

$n=10$

Из таблицы критериев Стьюдента:

$t_{\alpha/2}=t_{0.025}=2.262$

Левая граница доверительного интервала:

In [9]:
x_ - 2.262 * sigma / 10**0.5

6.267538255912426

Правая граница доверительного интервала:

In [10]:
x_ + 2.262 * sigma / 10**0.5

6.912461744087575

Таким образом, искомый доверительный интервал:

$\bigl[ 6.268 ; 6.912 \bigr]$

3. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.

$H_0: \mu = \mu_0$

$H_1: \mu > \mu_0$

Известно, что

$\mu_0 = 17$

$\overline{X} = 17.5$

$\alpha = 0.05$

$n = 100$

$\sigma^2 = 4$, значит $\sigma = 2$

Предположим, что в выборке из 100 шариков диаметр подчиняется нормальному закону распределения (согласно ЦПТ). Кроме того, дисперсия известна. Поэтому для проверки гипотезы будем использовать $z$-тест.

$Z_{н}=\frac{\overline{X}-\mu_0}{\sigma / \sqrt{n}}$

$Z_{н}=\frac{17.5-17}{2/10}=\frac{0.5}{0.2}=2.5$

Из $z$-таблиц находим расчетное значение:

$Z_{р}=Z_{\alpha}=Z_{0.05}=1.645$

$2.5>1.645$, значит $Z_{н}>Z{р}$ и нулевая гипотеза неверна с вероятностью ошибки 5%. Принимаем альтернативную гипотезу $H_1$.

4. Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет: 202, 203, 199, 197, 195, 201, 200, 204, 194, 190. Известно, что их веса распределены нормально. Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

$H_0: \mu = \mu_0$

$H_1: \mu \ne \mu_0$

Так как веса распредены нормально, а дисперсия генеральной совокупности неизвестна, то для проверки гипотезы будем использовать критерий Стьюдента и несмещенную оценку дисперсии из имеющейся выборки.

In [11]:
x = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])
x

array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])

In [12]:
x_ = x.mean()
x_

198.5

In [13]:
sigma = x.std(ddof=1)
sigma

4.453463071962462

Таким образом,

$\overline{X}=198.5$

$\mu_0=200$

$n=10$

$\sigma_{н}=4.45$

$\alpha=0.01$

$t_{н}=\frac{\overline{X}-\mu_0}{\sigma_{н} / \sqrt{n}}$

In [14]:
(x_ - 200) / (sigma / 10**0.5)

-1.0651074037450896

$t_{н}=-1.065$

Из $t$-таблиц находим расчетное значение:

$t_{р}=t_{\alpha/2}=t_{0.005}=3.250$

То есть область принятия нулевой гипотезы лежит между $t_{н}=-3.250$ и $t_{н}=3.250$.

$-3.250<-1.065<3.250$, значит нулевая гипотеза верна с доверительной вероятностью 99%.